README.md

```diff
@@ -1,4 +1,5 @@
-[![Build Status](https://travis-ci.org/CICE-Consortium/Icepack.svg?branch=master)](https://travis-ci.org/CICE-Consortium/Icepack)
+[![Build Status](https://travis-ci.com/CICE-Consortium/Icepack.svg?branch=master)](https://travis-ci.com/CICE-Consortium/Icepack)
+[![GithubActionsIcepackTesting](https://github.com/Apcraig/Icepack/workflows/GithubActionsIcepackTesting/badge.svg)](https://github.com/Apcraig/Icepack/actions)
```

Comment:
Maybe we could rename the Travis CI bagde too "Travis-CI", and the new GitHub actions to GH Actions, which would be cleaner at the top of the README. And why not use the right URL from the start, so that we don't have to modify it after the fact. Also, we could link directly to the master branch, and the "push" events on the Actions page:

```suggestion
[![Travis-CI](https://travis-ci.com/CICE-Consortium/Icepack.svg?branch=master)](https://travis-ci.com/CICE-Consortium/Icepack)
[![GH Actions](https://github.com/CICE-Consortium/Icepack/workflows/GithubActionsIcepackTesting/badge.svg)](https://github.com/CICE-Consortium/Icepack/actions?query=branch%3Amaster+event%3Apush)
```

Also if we rename the worflow as I suggest above, we should correspondingly change the "GithubActionsIcepackTesting" part.

Reply:
Again, for testing purposes I had to point to test something that existed.  I think this might be a multi-step process.  I'm still not sure why the PR is not creating github actions to trigger in the Consortium repo.  At this point, I'm speculating that this has to be committed before it will trigger it.  Again, I have it working in my repo on my branch, but we still may have to make a few PRs before it works in general, especially for PR pulls.

Reply:
Yes, it won't activate in the Consortium repo before it's merged.